modules/redis/redis.c

```diff
@@ -154,7 +191,11 @@ send_redis_command(RedisDriver *self, const char *format, ...)
 
   gboolean retval = reply && (reply->type != REDIS_REPLY_ERROR);
   if (reply)
-    freeReplyObject(reply);
+    {
+      if (trace_flag)
+        _trace_reply_message(reply);
```

Comment:
I think you could move the conditional inside the _trace_reply_message() function.